bundles/org.openhab.ui.homebuilder/web/src/fieldMultiselect.vue

```diff
@@ -1,169 +0,0 @@
-<template>
```

Comment:
Nevermind I will revert this :-)

This handled the icon support it was kind of a fork